onnxscript/utils/graph_view_utils.py

```diff
@@ -0,0 +1,263 @@
+from onnxscript import ir
+
+import ast
+
+
+def is_initializer(value):
+    return input.producer() == None
```

Comment:
When a value is a graph input, its producer is also None. I will add a method to make this check more robust

Reply:
https://github.com/microsoft/onnxscript/pull/2282